utils/utils.go

```diff
@@ -67,12 +66,6 @@ func GetKubeClient() *kubernetes.Clientset {
 	return clientSet
 }
 
-// ToISO8601DateTimeString converts dateTime to ISO8601 Format
-// ISO8601 Format: 2020-01-01T01:01:01.10101Z.
-func ToISO8601DateTimeString(dateTime time.Time) string {
```

Comment:
Reviewers: this method was completely unused within Dapr (I noticed it because I was wondering if it should have been moved to dapr/kit like the other methods)